adapters/deepintent/deepintenttest/exemplary/simple-banner.json

```diff
@@ -0,0 +1,138 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+        {
+            "id": "test-imp-id",
+            "banner": {
+                "format": [
+                    {
+                        "w": 300,
+                        "h": 250
+                    },
+                    {
+                        "w": 300,
+                        "h": 600
+                    }
+                ]
+            },
+            "tagid": "16",
+            "displaymanager": "di_prebid",
+            "displaymanagerver": "2.0.0",
+            "ext": {
+                "bidder": {
+                    "TagID": "16"
```

Comment:
It's common for json variables to be lower case. You might want to consider "tagid" instead.

Reply:
using tagId as standard camelcase.

Reply:
Ok. Please verify it's written as `tagId` in the json tests and in the bidder params json file.